### PR TITLE
Exp/universal fetch

### PR DESCRIPTION
**Pull request by NSeydoux**

### CODE REVIEW

File: jest.setup.ts [hunk 1]
```diff
@@ -20,3 +20,16 @@
...
+if (
+  typeof globalThis.Response === "undefined" ||
+  typeof globalThis.Request === "undefined" ||
+  typeof globalThis.Headers === "undefined"
+) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  globalThis.Response = Response as any;
+  globalThis.Request = Request as any;
+  globalThis.Headers = Headers as any;
+  globalThis.fetch = Headers as any;
```
Comment (jeswr) [inline jest.setup.ts:34]:
```suggestion
  globalThis.fetch = fetch as any;
```

Comment (jeswr) [inline jest.setup.ts:34]: This seems like it could be a utility in universal fetch?

**Review by jeswr — CHANGES_REQUESTED** [review]

File: jest.setup.ts (hunk 1 above)
Comment (NSeydoux) [reply jest.setup.ts:34]: I was thinking about having it as part of `@inrupt/jest-jsdom-polyfills`. I should have been more explicit about this, but this PR (which should have been a draft) isn't meant to be merged directly, it's just part of an overall experiment I'm running for Node 18 support across our different libraries.

Comment (NSeydoux) [reply jest.setup.ts]: Good catch

**Review by chelseapinka — APPROVED** [review]

**Pull request merged**